src/main/java/com/dj/core/model/graph/ConnectedNeuron.java

```diff
@@ -2,18 +2,29 @@
 
 import com.dj.core.model.activation.ActivationFunction;
 import com.google.common.util.concurrent.AtomicDouble;
+import org.apache.commons.math3.linear.*;
 
 import java.util.*;
 
 public class ConnectedNeuron implements Neuron {
 
+    private final static int VECTOR_ROWS_COUNT = 1;
+    private final static int VECTOR_ROW_INDEX = 0;
```

Comment:
please either use \n between fields here or remove \n in between other fields.

Reply:
Added \n